junit-runtime/src/main/scala/org/hamcrest/CoreMatchers.scala

```diff
@@ -1,117 +1 @@
-/*
- * Ported from https://github.com/hamcrest/JavaHamcrest/
- */
-package org.hamcrest
 
-import org.hamcrest.core._
-
-object CoreMatchers {
```

Comment:
What happened here?

Reply:
Not quite sure of it. This happened after I added a top-level `scalafmtAll` task key and ran it. My suspicion is that this may be the result of a bug in the scalafmt sbt plugin: if multiple sbt subprojects depend on the same files, it's possible that different scalafmt threads overwrite each other, and that this is what we're seeing here.

I'll go ask in the scalafmt gitter, but if it indeed is a bug with scalafmt, or if scalafmt doesn't support this style of top-level tasks, I think we may have to delay this PR a little.